traffic_ops_ort/traffic_ops_ort.pl

```diff
@@ -0,0 +1,2662 @@
+#!/usr/bin/perl
```

Comment:
Looks like you're necromancing, which while not strictly illegal has been banned guild-wide in all nine provinces by Arch Mage Hannibal Traven since 3E 431.

Best practice is to not practice necromancy.

Reply:
Removed in 210a4027d3. There's a cache configs bug I have yet to report and I used `traffic_ops_ort.pl` in CDN in a Box (which works with the `uname` shim still there) to rule it being a regression from `traffic_ops_ort.pl` to `t3c`.